odl/tomo/data/clintec_spect.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Helpers for reading CLINTEC SPECT data formats.
+
+For the bilinear relationship of linear attenuation coefficients and
+Houndsfield units see, for example
+
+Investigation of the relationship between linear attenuation
+coefficients and CT Houndsfield units using radionuclides for SPECT
+S. Brown et al, Applied Radiation and Isotopes 66 (2008) 1206-1212
+
+and references therein.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import dicom
```

Comment:
This will fail on default installations since the `dicom` dependency is missing. Add this to the `setup.py` `extra_dependencies` under a name like `dicom`.